gather.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+// Gatherer returns the metric gatherer
+func Gatherer() prometheus.Gatherer {
```

Comment:
Ok, so now that these are exposed, how do you see us using them in the projects?  Right now everything that is in this package is being used.  There are no unused public APIs so I'm just trying to get an idea of how we are going to use this and the below function in our projects.